Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -974,3 +975,15 @@ spec:
     user's responsibility.
 
 The operator does not unset any removed config options, it is the user's responsibility to unset or set the default value for each removed option manually using the Ceph CLI.
+
+## CSI Driver Options
+
+The CSI driver options mentioned here are applied per Ceph cluster. The following options are available:
```

Comment:
per Ceph cluster(external or converged)

Reply:
I think no need to mention external/converged. There is nothing special for csi settings between those two modes.